src/plugins/intel_cpu/src/nodes/scatter_update.h

```diff
@@ -16,6 +17,8 @@ enum class ScatterUpdateMode {
     ScatterElementsUpdate
 };
 
+class ReduceMean;
```

Comment:
No the best idea to have such a generic name inside common plugin namespace. It should have SEU specific name (like ScatterUpdateMode) or hidded inside namespace/class

Reply:
done with namespace for SEU